inscriptions/ISic0938.xml

```diff
@@ -170,17 +175,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140419">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Θέδουλο|ς<lb n="2"/>ἐνθάδε<lb n="3"/>κῖτε.
-                    <lb n="4"/>
+                    <lb n="1"/>Θέδου<choice><corr>λ</corr><sic>χ</sic></choice>ο<supplied reason="undefined">ς</supplied> <!-- littera ς scripta fin. v.2 -->
```

Comment:
Again, I don't think it should be with 'supplied', because it is there, just out of place. Leave unmarked, but add note in the apparatus.